uarray/_uarray_dispatch.cxx

```diff
@@ -852,12 +933,12 @@ PyTypeObject FunctionType = {
    | Py_TPFLAGS_HAVE_GC),         /* tp_flags */
   0,                              /* tp_doc */
   (traverseproc)Function_traverse,/* tp_traverse */
-  0,                              /* tp_clear */
+  (inquiry)Function_clear,        /* tp_clear */
```

Comment:
Is it possible to move the `Function_*` methods to be static `Function::*` methods?

Reply:
(I'm more of an OOP person myself. 😄)